### ISSUE TITLE
Prototyp 

### ISSUE THREAD

**Issue by Joernzie** [spochtfreund/mowIT#4]

**Comment by Joernzie** [comment]
Gleichzeitig mit den Kosten musst auch geklärt werden was wir mit dem Prototypen machen.  Ich habe nämlich großes Interesse daran deb Prototyp oder einen Clon möglichst schnell in meinem Garten zu verwenden. Somit bin ich natürlich auch bereit einen größen Teil der Kosten zu tragen. Deshalb ist auch die Modularität so wichtig, damit wir schnell weitere Exemplare bauen können.

**Comment by Joernzie** [comment]
Genau das meinte ich mit Modularität.  Einzelne Radmodule und ein Sensor-Steuermodul. Alles zum einfach anstecken.

**Comment by Biggs81** [comment]
Habe nicht verstanden was wir in diesem Ticket machen wollen?

**Comment by Joernzie** [comment]
Diskutieren wo wier den Prototyp aufbauen und wie viele bzw. frühzeitig entscheiden wo der Prototyp zum Einsatz kommt.

**Comment by Biggs81** [comment]
Ah. 

Ich bin dafür als erstes mal deinen Honda zu pimpen. 

Wäre dafür dass wir drei Testszenarien definieren
* Szenario "J" in Winsen
* Szenario "H" in Meine
* Szenario "B" in Bissendorf

Anforderung wäre, dass das System in der Lage ist diese drei Szenarien potentiell zu packen.

**Comment by Joernzie** [comment]
Aus Fuhrberg wird Winsen. Da wohnen wir seit 2014.

**Comment by Biggs81** [comment]
Naaaaa guut

**Comment by Biggs81** [comment]
@spochtfreund?

**Comment by spochtfreund** [comment]
Jo. passt. Aber hier in Meine ist der Rasen immer gemäht. Könne ihn aber gerne auch hier laufen lassen...
